app/components/features/features.styl

```diff
@@ -1,5 +1,6 @@
 // @import '../../variables.css'
```

Comment:
I run `npm start`, it isn't even close to the design. Pls don't merge into `master` unfinished work
Thanks